src/cli/self_update.rs

```diff
@@ -433,10 +433,10 @@ pub(crate) fn install(
         let msg = pre_install_msg(opts.no_modify_path)?;
 
         md(&mut term, msg);
-
+        let mut customised_install: bool = false;
```

Comment:
Please omit the type annotation, which is unnecessary, and rename `customised_install` to `customized_install` to match convention for US English spelling.

Reply:
this has become a enum now. this seems to be outdated

Reply:
will change the spelling though